apps/website/src/pages/docs/footer.mdx

```diff
@@ -14,14 +14,48 @@ import * as AllExamples from 'examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-footer' />
-
 <LiveExample src='Footer.main.tsx' truncate={false}>
   <AllExamples.FooterMainExample client:load />
 </LiveExample>
 
 The footer is fixed to the bottom of a web page, and contains your typical legally required links. The legal footer is only required to be displayed on pages that are considered a “main entry point into your application”. This means that the sign in and home page of any hub / app are good places to display the footer. You do not need to display the footer within an App UI widget, or deeper pages within an app. You can not include the footer on a page with infinite scrolling because you can not reach the bottom of the page.
 
+## Customizations
+
+Alter standard footer section by modifying elements or adding new custom elements and content.
+
+### Appended Custom Elements
+
+Add custom element by passing `customElements` prop to Footer component, set the `title` property as label for the custom link and `url` property for URL associated with the custom link.
+
+<LiveExample src='Footer.appendedcustomelements.tsx'>
+  <AllExamples.FooterAppendCustomElementsExample client:load />
+</LiveExample>
+
+### Only Custom Elements
+
+Add only custom elements by passing `customElements` prop to Footer component. It expects an array of objects each describing a custom link for the footer. Set the `title` and `url` property for each defined custom element.
```

Comment:
> Set the `title` and `url` property for each defined custom element.

But the example below this paragraph contains elements with just the `title` property. So maybe there should be a mention that the `url` is optional; and if not passed, it will render as plain text and not an anchor.

Reply:
I think it kind of makes sense that it is optional because the first element in the array has both title and URL and the next one has only title set. I am not sure if I should add it or not. I can add it if you would like.

Reply:
I mean to say that the doc website says, "Set the `title` and `url` property for each defined custom element.". i.e. set both `title` and `url`. But the example below has some elements only with `title` and no `url`. So I was thinking the last sentence on L37 needs to be rephrased.

Reply:
Fixed